src/renderer/redux/selectors/media.js

```diff
@@ -0,0 +1,15 @@
+import * as settings from "constants/settings";
```

Comment:
I noticed that `/selectors/video.js` has not been deleted. Are we still using it?